app/assets/javascripts/host_checkbox.js

```diff
@@ -63,7 +63,7 @@ $(function() {
 });
 
 function removeForemanHostsCookie() {
-  $.cookie($.cookieName, null);
+  $.removeCookie($.cookieName, { path: '/' });
```

Comment:
We're relying on defaults, which sets the path to '/'. 
I have removed the ```path: '/'``` as it is already set to default.
Thanks